src/node_i18n.h

```diff
@@ -38,7 +38,7 @@
 namespace node {
 namespace i18n {
 
-bool InitializeICUDirectory(const std::string& path);
+bool InitializeICUDirectory(const std::string& path, std::string* error);
```

Comment:
Just curious, why `std::string*` and not `const std::string&`?

Reply:
This is an out parameter, the caller gets the error string if it fails